src/HttpTrigger/sample.dat

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Unless you're using this file, you can delete it.

Reply:
deleted the file.